doc/tutorials/ios-app.md

```diff
@@ -270,30 +277,95 @@ output will appear similar to the following:
 ```bash
 INFO: Found 1 target...
 Target //ios-app:ios-app up-to-date:
-  bazel-bin/ios-app/ios-app.ipa
-INFO: Elapsed time: 0.565s, Critical Path: 0.44s
+  bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-4e6c2a19403f/bin/ios-app/ios-app.ipa
+INFO: Elapsed time: 0.141s, Critical Path: 0.00s
 ```
 
 ### Find the build outputs
 
 The `.ipa` file and other outputs are located in the
-`$WORKSPACE/bazel-bin/ios-app` directory.
+`$WORKSPACE/bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-4e6c2a19403f/bin/ios-app/ios-app.ipa` directory.
```

Comment:
```suggestion
`$BUILD_WORKSPACE_DIRECTORY/bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-4e6c2a19403f/bin/ios-app/ios-app.ipa` directory.
```

Reply:
Here `$WORKSPACE` is referencing the custom ENV variable set at the beginning of the tutorial, not the Bazel's specific one. I guess they're equivalent, but shouldn't we use the same one for consistency?

Reply:
Ahh, true.